content/v2.0/influx-templates/create.md

```diff
@@ -1,21 +1,105 @@
+**InfluxDB OSS** supports multiple organizations.
+In **InfluxDB Cloud**, your user account is an organization.
+**We recommend using InfluxDB OSS to create Influx templates.**
+{{% /note %}}
+
+**To create a template:**
+
+1. [Start InfluxDB](/v2.0/get-started/).
+2. [Create a new organization](/v2.0/organizations/create-org/).
+3. Use the InfluxDB UI to add resources to your template.
+   The following resources are templatable:
+
+   - [buckets](/v2.0/organizations/buckets/create-bucket/)
+   - [checks](/v2.0/monitor-alert/checks/create/)
+   - [dashboards](/v2.0/visualize-data/dashboards/create-dashboard/)
+   - [dashboard variables](/v2.0/visualize-data/variables/create-variable/)
+   - [labels](/v2.0/visualize-data/labels/)
+   - [notification endpoints](/v2.0/monitor-alert/notification-endpoints/create/)
+   - [notification rules](/v2.0/monitor-alert/notification-rules/create/)
+   - [tasks](/v2.0/process-data/manage-tasks/create-task/)
+   - [Telegraf configurations](/v2.0/write-data/use-telegraf/)
+
+4. Export the template _(see [below](#export-a-template))_.
+
+## Export a template
+There are two methods for exporting a template.
+
+1. Export all resources in an organization _(recommended)_
+2. Export specific resources in an organization
+
+### Export all resources
+Use the `influx pkg export all` command to export all templatable resources
+into a new template manifest.
+Provide the following:
+
+- **Organization name** or **ID**
+- **Authentication token** with read access to the organization
+- **Destination file** path for the template manifest.
+  The extension of the destination file determines the output format of the manifest file.
+  Export template manifests in either YAML (`.yml` or `.yaml`) or JSON (`.json`).
+
+###### Export all resources to a template manifest
+```sh
+# Syntax
+influx pkg export all -o <org-name> -f <filepath> -t <token>
+
+# Example
+influx pkg export all \
+  -o my-org \
+  -f ~/templates/awesome-template.yml \
+  -t $INFLUX_TOKEN
+```
+
+For information about other options and flags, see the
+[`influx pkg export all` documentation](/v2.0/reference/cli/influx/pkg/export/all/).
+
+### Export specific resources
+Use the `influx pkg export` with resource flags to export all specific templatable
+resources into a new template manifest.
+Provide the following:
+
+- **Organization name** or **ID**
+- **Authentication token** with read access to the organization
+- **Destination file** path for the template manifest.
+  The extension of the destination file determines the output format of the manifest file.
+  Export template manifests in either YAML (`.yml` or `.yaml`) or JSON (`.json`).
+- **Resource flags** with corresponding lists of resource IDs to include in the template.
+  For information about what resource flags are available, see the
+  [`influx pkg export` documentation](/v2.0/reference/cli/influx/pkg/export/).
+
+###### Export specific resources to a template manifest
```

Comment:
Export specific resources to a template

Reply:
can you export a specific resource to a template after a template is created? or is exporting effectively the same as creating a template?

Reply:
Exporting is the same as creating a template. It will generate an entirely new manifest. It won't extend and already existing manifest. This is a really cumbersome way to create a template and is why we recommend using OSS to build templates within an org meant only for building the template... so they don't have to use this method.